### DISCUSSION TITLE
manjaro 不能在google chrome中输入中文，firefox和终端中没有此问题。

### DISCUSSION THREAD

**Discussion by xclidongbo** [fcitx/fcitx5#651] [Q&A]
不能在google chrome中输入中文，firefox和终端中没有此问题。

**Comment by bickman** [comment]
你解决了没有,我的archlinux+gnome桌面也无法在chromium里输入中文,其他软件没问题,我都是在终端里输入中文在复制进来的

**Reply by bickman** [reply]
换成dwm又可以在chromium里输入中文了,貌似和wayland有关,gnome默认用的wayland

**Reply by DyeMonarch** [reply]
我就是在dwm中也无法输入中文。有点烦

**Comment by DyeMonarch** [comment]
同使用debian＋dwm，也是。使用Firefox可以。chrome，edge，vscode都不可以唤出输入法
